bundles/io/org.eclipse.smarthome.io.rest.persistence/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,18 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Eclipse SmartHome Core Persistence Rest
+Bundle-SymbolicName: org.eclipse.smarthome.io.rest.persistence
+Bundle-Version: 0.8.0.qualifier
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Import-Package: org.eclipse.smarthome.core.items,
```

Comment:
Can you use alphabetical order for all imports?
That is not a must, but could improve readability.